CHANGELOG.md

```diff
@@ -1,5 +1,92 @@
 # Changelog
 
+## Pelion Edge 2.2 - January 2021
```

Comment:
Adding a different note. @ygoyal18 this file should include the change for the hotifx that was needed a week or two ago: https://github.com/PelionIoT/meta-pelion-edge/commit/a033908508908dae82d0c746e958aa3e9f44723b

Reply:
Sure. Will add that.